lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -202,7 +202,7 @@ def test_persistent_grades_not_enabled_on_course(self, default_store):
             self.set_up_course(enable_persistent_grades=False)
             self.assertFalse(PersistentGradesEnabledFlag.feature_enabled(self.course.id))
             with check_mongo_calls(0):
-                with self.assertNumQueries(0):
+                with self.assertNumQueries(1):
```

Comment:
The additional `datetime_resolved IS NONE` clause shouldn't mess with our indexing, since the task_id is a UUID which ought to be unique (but isn't always, only due to the task duplication issue).